### PR TITLE
wip - Ensure that all scorecard tests are checked and with changes made in the PR

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
- all e2e test with are using the dev tag image
- add test:dev scorecard images used to be built in the CI
- add custom test for ansible/helm to ensure that this test will work with them
- cleanup scorecard and ensure that these images and binaries will be generated in the place and at the same way ansible/helm. see:
Screen Shot 2020-09-04 at 18 32 15

- fix targets to be able to generate the binaries that are copied to the images on them and in this way check the PR with the changes performed. 
- remove the old bin/entrypoint standard in favour of the scorecard follow up the same ansible/helm approach to define the user/dir that should be used. 

**Motivation for the change:**
- Closes: #3839

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -284,16 +284,16 @@ test-subcommand-olm-install:
 
 test-e2e: test-e2e-go test-e2e-ansible test-e2e-ansible-molecule test-e2e-helm ## Run the e2e tests
 
-test-e2e-go:
+test-e2e-go: image-build-scorecard-test image-build-custom-scorecard-tests
```
Comment (jmccormick2001) [inline Makefile:287]: probably will want to go ahead and add the scorecard-test-kuttl image while you are in there.

Comment (camilamacedo86) [reply Makefile:287]: I did not saw this image used in the scaffolds. Are we scaffolding any manifests with the scorecard-test-kuttl? 
if not, then I do not think that we need to gen for the tests.

Comment (camilamacedo86) [reply Makefile:323]: done

**Comment by camilamacedo86** [comment]
The issue was solved in another pr. 
Closing this one.

**Pull request closed without merging**